app/views/admin/permissions/index.html.haml

```diff
@@ -34,30 +34,32 @@
     %tr
       %th= User.model_name.human
       - @available_roles.each do |role|
-        %th= role.to_s.humanize
+        %th{:title => t(".role_description.#{role}")}
+          = role.to_s.humanize
+      %th
   %tbody
-    - User.this_tenant.with_any_role(*@available_roles).each do |user|
+    %fieldset.form__fieldset
       %tr
-        %th= user.to_s_with_email
+        %th
+          = form_tag(add_role_permissions_path, { :method => :put, :id => "add_permissions_form"})
+          = select_tag :users_id, options_from_collection_for_select(User.this_tenant.all, "id", "to_s_with_email"), multiple: true, class: "chosen-select"
+        - @available_roles.each do |role|
+          %td{:title => t(".role_description.#{role}")}
+            = check_box_tag 'roles_names[]', role, form: "add_permissions_form"
+        %td
+          = submit_tag t(".add_selected_permissions"), class: "button", form: "add_permissions_form"
+    - User.this_tenant.with_any_role(*@available_roles).each do |user|
+      %tr{id: (user.id == current_user.id ? 'current_user_row' : nil)}
+        %th{title: (user.id == current_user.id ? t(".warn_modify_your_account") : nil)}
+          = user.to_s_with_email
         - @available_roles.each do |role|
-          %td
-            = user.has_role?(role)
-            - if user.has_role?(role)
-              = link_to "X", set_role_permissions_path(role_name: role, user_id: user.id), method: :put, data: { confirm: t("are_you_sure") }
+          %td{:title => t(".role_description.#{role}")}
+            = link_to (check_box_tag 'role_name', checked: user.has_role?(role)), set_role_permissions_path(roles_names: [role], users_id: [user.id]), method: :put
```

Comment:
this is really confusing. Is it a link, or a checkbox? Why would it be both? If it's a link, which performs an action when you click on the checkbox, then remove the checkbox, and put a button here instead. (I dislike "links" which perform "actions". Links should be GET, buttons can be PUT/POST/DELETE.